func_test_settings.py

```diff
@@ -0,0 +1 @@
+DATABASES = {"default": {"NAME": "pulp", "USER": "pulp", "PASSWORD": "pulp"}}
```

Comment:
Can this be moved out of the root dir? Maybe into pulp_ansible/tests/assets or something?